discovery-provider/nginx_conf/main.lua

```diff
@@ -13,6 +13,43 @@ math.randomseed(string.byte(resty_random.bytes(1)))
 
 local _M = {}
 
+local redirect_target_weights = {}
+for index, redirect_target in ipairs(config.redirect_targets) do
+    redirect_target_weights[redirect_target] = 1
+end
+
+function update_redirect_target_weights (premature)
```

Comment:
is this `premature` input used in code? didn't see it in this PR

Reply:
https://github.com/AudiusProject/audius-protocol/pull/2142/files#